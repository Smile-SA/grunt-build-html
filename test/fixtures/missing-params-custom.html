<%= include('missing-params', {
  additionalParam: 'custom param'
}) %>