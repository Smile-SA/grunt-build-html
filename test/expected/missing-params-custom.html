<!DOCTYPE html>
<html>
  <head>
    <title>My title</title>
  </head>
  <body>
    Hello world - with param : custom param.
  </body>
</html>
